.eslintrc.js

```diff
@@ -72,7 +72,7 @@ module.exports = {
     'import/named': 'warn',
     'import/no-default-export': 'error',
     'import/no-named-as-default': 'off',
-    'jest/expect-expect': 'warn',
```

Comment:
I don't like the sound of turning this rule off completely - what was this turned off for?

Reply:
This rule gives a warning when at it block doesn't contain an expect. We turned it off because the entire block was being highlighted as the tests were being written.

Reply:
Couldn't one just stub out an expect instead to get rid of the highlighting? I generally don't encourage disabling config like this, especially if it's recommended by the plugin developers, as it is here: https://github.com/jest-community/eslint-plugin-jest#rules This affects our branch protection lint job too, from what I can tell.

Reply:
There's a few ways we might be able to address this.

Refactoring card added at https://trello.com/c/d4GRF0Oy/858-re-enable-jest-expect-expect-rule